edb/server/compiler/errormech.py

```diff
@@ -374,6 +374,16 @@ def _static_interpret_cardinality_violation(_code, err_details):
     return errors.InternalServerError(err_details.message)
 
 
+@static_interpret_by_code.register(
+    pgerrors.ERROR_DATA_EXCEPTION)
+def _static_interpret_data_exception(_code, err_details):
+    # Problem with range bounds
+    if err_details.message.startswith('range lower bound must be'):
+        return errors.InvalidValueError(err_details.message)
+
+    return errors.InternalServerError(err_details.message)
```

Comment:
It's probably safe to simply translate all data exceptions to `InvalidValue`.

Reply:
Fair point